dist/templates/ovnkube-single-node-zone.yaml.j2

```diff
@@ -299,6 +301,10 @@ spec:
           name: run-systemd
           subPath: private
           readOnly: true
+        {% if enable_ipsec=="true" -%}
+        - mountPath: /var/run/pluto
```

Comment:
Does that path exist upstream? What's this path for ?

Reply:
This is to get access to `/var/run/pluto/pluto.ctl` UDS file from `ovnkube-controller` container to run ipsec command to retrieve states from pluto process.